hypervisor/boot/sbl/multiboot.c

```diff
@@ -54,7 +54,7 @@ static void parse_other_modules(struct acrn_vm *vm, const struct multiboot_modul
 			static int32_t copy_once = 1;
 
 			start = end + 1; /*it is fw name for boot args */
-			snprintf(dyn_bootargs, 100U, " %s=0x%x@0x%x ", start, mod_size, mod_addr);
+			snprintf(dyn_bootargs, 100U, " %s=0x%x@0x%x ", start, mod_size, (uint64_t)mod_addr);
```

Comment:
are  u using old code base? I didn't see this

Reply:
I am using master branch. (git clone https://github.com/projectacrn/acrn-hypervisor.git)

Reply:
Then, should we use %p？ or, if we use mod_addr, should we use %lx ?

Reply:
@dongyaozu used %p, thats better.